smart_card_connector_app/src/window-devices-displaying.js

```diff
@@ -116,6 +119,15 @@ function makeReaderNameForDisplaying(readerName) {
   return readerName;
 }
 
+function updateAddDeviceButtonText(readersCount){
+  if(readersCount == 0){
+    addDeviceElement.dataset.i18n = 'addDevice';
+  }
+  else{
```

Comment:
nit: put these in one line and with a space between a bracket and "else":
"} else {"